packages/discord.js/src/structures/interfaces/TextBasedChannel.js

```diff
@@ -78,6 +78,7 @@ class TextBasedChannel {
    * @typedef {BaseMessageOptions} BaseMessageCreateOptions
    * @property {boolean} [tts=false] Whether the message should be spoken aloud
    * @property {string} [nonce=''] The nonce for the message
+   * @property {boolean} [enforceNonce=false] Whether the message should check for uniqueness
```

Comment:
It's not being defaulted to anything in discord.js, so I wouldn't put `=false` here.

```suggestion
   * @property {boolean} [enforceNonce] Whether the message should check for uniqueness
```